clients/python/tests/test_api.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import schemathesis
+from hypothesis import HealthCheck, settings
+
+schema = schemathesis.pytest.from_fixture("generated_schema")
+
+
+@schema.parametrize()
+@settings(
+    deadline=None,
+    suppress_health_check=[
+        HealthCheck.filter_too_much,
+    ]
+)
+@pytest.mark.e2e
+def test_mr_api_stateless(setup_env_user_token, case):
+    """Test the Model Registry API endpoints.
+
+    This test uses schemathesis to generate and validate API requests
+    """
+
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {setup_env_user_token}"
+    }
+
+    # We need to ensure valid data
+    if case.path == "/api/model_registry/v1alpha3/model_versions/{modelversionId}/artifacts":
+        case.path_parameters["modelversionId"] = "test-model-version-1"
+
+        if case.method == "POST":
+            case.body = {
+                "artifactType": "model-artifact",
+                "name": "test-artifact",
+                "uri": "s3://test-bucket/test-artifact",
+                "metadata": {}
+            }
```

Comment:
I would need to double check the internal logic of the library again since it's been some time, but I think that this is not needed. 
From the openAPI schema the library should be able to understand that these fields are required in the request it sends to the endpoint. 
If the fields are not set to required, or if the library somehow does not pick it up automatically, we might need to either revisit the schema or the implementation of the library. 
Could you please double check what happens if these lines are not used?

If instead you are trying to force specific values here, IMHO for the _stateless_ test it is not needed as we are effectively fuzzing the API with random data, and I'd instead suggest implementing this (and additional logic) in the _stateful_ test.